zcncore/wallet.go

```diff
@@ -244,7 +244,7 @@ func GetMinShardersVerify() int {
 
 func getMinShardersVerify() int {
 	minSharders := util.MaxInt(calculateMinRequired(float64(_config.chain.MinConfirmation), float64(len(_config.chain.Sharders))/100), 1)
-	Logger.Info("Minimum sharders used for verify :", minSharders)
+	// Logger.Info("Minimum sharders used for verify :", minSharders)
```

Comment:
why it is removed?

Reply:
if we enabled this, the wallet_test.go get stuck, so when debugging the test code, I remove logger and test worked again. Other Logger.Info is find, I'm not sure why it get stuck with this function.

Reply:
it is so strange. It should have a corner issue. Logger.Info should be not the bug. Could you try to test it more, and find what happens?

Reply:
@Rizary I guess bcz you need to init Logger in tests, but we should not change main code

Reply:
The problem is not that apparently. It seems that `_config.chain.MinConfirmation` is still `nil` even though I already run `InitStorageSDK` which also run `zcncore.Init()`. In general, it should get updated. I'm still looking for solution.